backends/module.mk

```diff
@@ -146,9 +146,18 @@ MODULE_OBJS += \
 	graphics/surfacesdl/surfacesdl-graphics.o \
 	mixer/sdl/sdl-mixer.o \
 	mutex/sdl/sdl-mutex.o \
-	plugins/sdl/sdl-provider.o \
 	timer/sdl/sdl-timer.o
 
+ifdef KOLIBRIOS
+MODULE_OBJS += plugins/kolibrios/kolibrios-provider.o
```

Comment:
This can be combined with the other `ifdef KOLIBRIOS` check.

Reply:
Done